src/app/components/Billboard/index.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import Billboard from '.';
+import { StoryProps, StoryArgs } from '../../models/types/storybook';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+interface Props extends StoryProps {
+  text?: string;
+  longText?: string;
+  showLiveLabel?: boolean;
+}
+
+const Component = ({
+  text = '',
+  longText = '',
+  showLiveLabel = false,
+}: Props) => {
+  return (
+    <Billboard
+      heading={text}
+      description={longText}
+      link="https://www.bbc.co.uk/ws/languages"
+      image="https://ichef.bbci.co.uk/ace/standard/{width}/cpsdevpb/1d5b/test/5f969ec0-c4d8-11ed-8319-9b394d8ed0dd.jpg"
+      showLiveLabel={showLiveLabel}
+      altText="alt text"
+    />
+  );
+};
+
+export default {
+  title: 'Components/Billboard',
+  Component,
+  parameters: {
+    docs: { readme },
+    metadata,
+  },
```

Comment:
Definitely not needed, but you could maybe try out the new `args` and `argTypes` control system here to make the LiveLabel a toggle-able thing and have the 1 story? Although having them as separate stories makes it easier to share with others.

Reply:
I guess we want to have 2 stories so that we a separate visual snapshot for a billboard with and without a live label. This will be necessary when we have to put the heading text below the live pulse + label as part of https://jira.dev.bbc.co.uk/browse/WSTEAMA-1081.

But thanks for the suggestion - we will consider this in future components!